src/StructArrays.jl

```diff
@@ -29,4 +29,12 @@ end
 import Adapt
 Adapt.adapt_structure(to, s::StructArray) = replace_storage(x->Adapt.adapt(to, x), s)
 
+# for GPU broadcast
+import GPUArraysCore: backend
+function backend(::Type{T}) where {T<:StructArray}
```

Comment:
```suggestion
import GPUArraysCore
function GPUArraysCore.backend(::Type{T}) where {T<:StructArray}
```

We prefer implicit imports in StructArrays for overloading.